src/ontology/cl-edit.owl

```diff
@@ -12936,11 +12937,13 @@ SubClassOf(obo:CL_0001030 obo:CL_0001060)
 
 # Class: obo:CL_0001031 (cerebellar granule cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:mah") obo:IAO_0000115 obo:CL_0001031 "Granule cell that is part of the cerebellum.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:mah") Annotation(oboInOwl:hasDbXref "doi:10.12688/f1000research.15021.1") Annotation(oboInOwl:hasDbXref "doi:10.3389/fncir.2020.611841") obo:IAO_0000115 obo:CL_0001031 "An excitatory granule cell with a soma located in the granular layer of cerebellar cortex. A mature cerebellar granule cell has short dendrites with a characteristic claw-like appearance and a long axon that ascends to the molecular layer where it bifurcates (except in non-teleost fish, where it does not bifurcate) and extends mediolaterally to form parallel fibers.")
```

Comment:
Consider adding 'granule cell of the cerebellum' as an exact synonym (doi:[10.3389/fncir.2020.611841](https://doi.org/10.3389/fncir.2020.611841))